chanzuckerberg/single-cell-data-portal
# feat: added suspension type filter. #2622

## Reason for Change
- #2622

## Changes
- Added suspension type filter to datasets and collections.

## Testing steps
- For both collections and datasets, verify:
  - suspension type filter is visible,
  - counts of suspension type filter values are correct, and,
  - selecting suspension type filter value filters result set correctly.
- Reviewed in rdev by @NoopDog and @hthomas-czi.
